exercises/ex_stam.py

```diff
@@ -0,0 +1,49 @@
+# Stam Exercise
+
+import sys
+PATH = 1
+FIRST_NAME = "Shira"
+LAST_NAME = "Vaknin"
+PATH_TO_OTHER_FILE = r"Z:\python\my_name.txt"
+
+
+def add_name(file_path, firs_name, last_name, other_path):
+    """
+    Get file address add my name to the line and save it in new location.
```

Comment:
The functions explanation should be in the same line as """
Your current sentence describes the function functionality instead of just what its purpose